src/components/layout/test/Panel-test.js

```diff
@@ -92,7 +92,7 @@ describe('Panel', () => {
       );
 
       assert.isAbove(
-        wrapper.find('div[data-component="CardContent"]').first().getDOMNode()
+        wrapper.find('div[data-component="CardContent"] p').first().getDOMNode()
```

Comment:
Because the `CardContent` in play here now has height-constraint classes applied to it, _it_ is height constrained in all cases but its children aren't, when scrolling is disabled.